distributions/LibreELEC/kernel_options

```diff
@@ -25,3 +25,33 @@ CONFIG_ASYMMETRIC_PUBLIC_KEY_SUBTYPE=y
 CONFIG_X509_CERTIFICATE_PARSER=y
 CONFIG_PKCS7_MESSAGE_PARSER=y
 CONFIG_PKCS8_PRIVATE_KEY_PARSER=m
+
+# packages/sysutils/systemd
+CONFIG_DEVTMPFS=y
+CONFIG_CGROUPS=y
+CONFIG_INOTIFY_USER=y
+CONFIG_SIGNALFD=y
+CONFIG_TIMERFD=y
+CONFIG_EPOLL=y
+CONFIG_NET=y
+CONFIG_SYSFS=y
+CONFIG_PROC_FS=y
+CONFIG_FHANDLE=y
+CONFIG_CRYPTO_USER_API_HASH=y
+CONFIG_CRYPTO_HMAC=y
+CONFIG_CRYPTO_SHA256=y
+CONFIG_SYSFS_DEPRECATED=n
+CONFIG_UEVENT_HELPER=n
```

Comment:
if I understand https://github.com/systemd/systemd/blob/main/README#L61 correctly, `CONFIG_UEVENT_HELPER` has to be set to "y" (since its necessary to have the option) `CONFIG_UEVENT_HELPER_PATH` which has to be set to `""` (empty string) see https://cateee.net/lkddb/web-lkddb/UEVENT_HELPER_PATH.html

Reply:
just use CONFIG_UEVENT_HELPER=n - the systemd docs are a bit outdated...